sdk/tracing/src/main/java/io/opentelemetry/sdk/trace/Sampler.java

```diff
@@ -77,5 +79,16 @@ SamplingResult shouldSample(
      *     {@link Decision#RECORD_AND_SAMPLE}.
      */
     Attributes getAttributes();
+
+    /**
+     * Return an optionally-updated {@link TraceState}, based on the parent TraceState. This may
+     * return the same {@link TraceState} that was provided originally, or an updated one.
+     *
+     * @param parentTraceState The TraceState from any parent span. Might be an empty TraceState, if
```

Comment:
We could document here that this will always be the same instance as the one passed to ShouldSample before.

Reply:
good idea. done!